mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,17 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
```

Comment:
Make sure the spacing is consistent with the rest of your codes, i.e. ``a, b, c`` instead of ``a,b,c``